exi-structures.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
Why is this header needed? Is there that many places where we need to include every single type header?

If it is needed, might as well update file name to be more consistent (ExiStructures.h maybe)

Reply:
Yea, fine to leave it if you want for the reason you mentioned. I prefer the explicitness of including just what you need, but prob not worth having to change things around.